src/Symfony/Bundle/FrameworkBundle/Tests/Controller/ControllerTest.php

```diff
@@ -37,10 +41,99 @@ public function testForward()
         $container->expects($this->at(0))->method('get')->will($this->returnValue($requestStack));
         $container->expects($this->at(1))->method('get')->will($this->returnValue($kernel));
 
-        $controller = new Controller();
+        $controller = new TestController();
```

Comment:
Can't this be handled with a magic `__call()` method in the `TestController` class?